### PR TITLE
Update repositories.txt - one URL changed and one URL removed

### PR DESCRIPTION
**Pull request by rei-vilo**
* Changed https://github.com/rei-vilo/PDLS_EXT3_Basic to https://github.com/rei-vilo/PDLS_EXT3_Basic_Global for consistency

* Removed https://github.com/rei-vilo/PDLS_EXT3_Basic_Fast_152 
The https://github.com/rei-vilo/PDLS_EXT3_Basic_Fast_152 is actually merged into https://github.com/rei-vilo/PDLS_EXT3_Basic_Fast

Thank you!

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @rei-vilo. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/11/29 15:54:58 Loaded 6413 libraries from DB
Changing URL of library PDLS_EXT3_Basic from https://github.com/rei-vilo/PDLS_EXT3_Basic.git to https://github.com/rei-vilo/PDLS_EXT3_Basic_Global.git
Success!
```
```
2023/11/29 15:56:44 Loaded 6413 libraries from DB
Removing PDLS_EXT3_Basic_Fast_152
Success!
```

**Comment by rei-vilo** [comment]
Thank you!

I guess the central repository would take some time to be synchronised.

Last release raised the error

>  ERROR: Library name PDLS_EXT3_Basic_Global not found in the Library Manager index. Library names are not allowed to change after being added to the index.     

`library.properties` lists

```
name=PDLS_EXT3_Basic_Global
```

**Comment by rei-vilo** [comment]
Done! Thank you for the pointer to the clear procedure.

**Pull request merged**
